staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2760,10 +2765,12 @@ type PodSpec struct {
 	// Value must be a positive integer.
 	// +optional
 	ActiveDeadlineSeconds *int64 `json:"activeDeadlineSeconds,omitempty" protobuf:"varint,5,opt,name=activeDeadlineSeconds"`
-	// Set DNS policy for containers within the pod.
-	// One of 'ClusterFirstWithHostNet', 'ClusterFirst' or 'Default'.
+	// Set DNS policy for the pod.
 	// Defaults to "ClusterFirst".
-	// To have DNS options set along with hostNetwork, you have to specify DNS policy explicitly to 'ClusterFirstWithHostNet'.
+	// Valid values are 'ClusterFirstWithHostNet', 'ClusterFirst', 'Default' or 'None'.
```

Comment:
This needs to indicate that `None` is controlled by a feature gate and by alpha.  It also needs to note that only 1.9 or later Kubernetes servers will allow the None value.

Reply:
Needs to be fixed for 1.9 so that public API documentation is correct.

Reply:
will file a PR to fix this.